build/tools/certgen.sh

```diff
@@ -28,6 +27,15 @@ ensureFolder() {
     fi
 }
 
+ensureCommand() {
+    if command -v "$1" >/dev/null 2>&1; then
+        #echo "$1 exists."
```

Comment:
```suggestion
        echo "$1 exists."
```
This message should be necessary :)